.github/workflows/stale.yml

```diff
@@ -1,35 +1,33 @@
-name: "Mark stale PRs & Issues"
+name: Mark stale PRs & Issues
 
 on:
   schedule:
-  # Run this every hour, so we are not spammed with changes at once. Later we could consider changing this to once a day.
-  - cron: "0 * * * *"
+  - cron: 0 0 * * *  # Runs at 00:00 UTC every day.
+
+permissions:
+  issues: write
+  pull-requests: write
 
 jobs:
-  stale-pr:
-    if: github.repository == 'ethereum/eips'
-    runs-on: ubuntu-latest
-    name: "Mark stale PRs"
-    steps:
-    - uses: actions/stale@v3
-      with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        days-before-stale: 60
-        days-before-close: 7
-        stale-pr-label: 'stale'
-        stale-pr-message: 'There has been no activity on this pull request for two months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.'
-        close-pr-message: 'This pull request was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback or request a review in a comment.'
-  stale-issue:
+  stale:
     if: github.repository == 'ethereum/eips'
     runs-on: ubuntu-latest
-    name: "Mark stale issues"
+    name: Mark stale PRs
     steps:
     - uses: actions/stale@v3
       with:
+        # General
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        days-before-stale: 180
-        days-before-close: 14
-        exempt-issue-labels: 'discussions-to'
-        stale-issue-label: 'stale'
-        stale-issue-message: 'There has been no activity on this issue for two months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.'
-        close-issue-message: 'This issue was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback or request a review in a comment.'
+        ascending: true # Since we have so many issues, the stale bot finds it hard to keep track. This makes sure that at least the oldest are removed.
+        # Issue config
+        stale-issue-message: There has been no activity on this issue in a month. It will be closed in a week if no further activity occurs
+        days-before-issue-stale: 30
+        days-before-issue-close: 7
+        exempt-issue-labels: discussions-to,long-term
+        stale-issue-label: stale
+        # PR config
+        stale-pr-message: There has been no activity on this pull request in a month. It will be closed in a week if no further activity occurs.
```

Comment:
Why did you remove the helper text that gives advice to the author on how to proceed?

Reply:
Re-added.